sample_packages/sample_python_package/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
```

Comment:
Can we use setup.py here so we can test catching code being executed on install too?

Reply:
Changed build system to use setuptools and added a setup.py script.